src/dmd/root/array.d

```diff
@@ -313,3 +313,15 @@ unittest
     array[2] = 910;
     assert([123, 421, 910, 123, 1, 2, 8, 20, 4, 3] == array.asDArray);
 }
+
+bool opEquals(T)(const(Array!T)* left, const(Array!T)* right)
```

Comment:
As always - new symbols:
- should have docstrings
- should use `ref` or `out` instead of pointers (except there's a good reason not to)

Reply:
since `left` and `right` could be null, using ref seems odd to me in this case